lib/presentation/profile/helper.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:wpa_app/app/injection.dart';
+import 'package:wpa_app/presentation/common/text_factory.dart';
+
+class HeaderWidget extends StatelessWidget {
```

Comment:
I don' think there is any added value of putting the header widget in a helper file. This can go into lib/presentation/profile/profile.dart